ipython/global_uncertainty.ipynb

```diff
@@ -192,14 +189,15 @@
    "outputs": [],
    "source": [
     "# Create ReactorPCEFactory global uncertainty analysis object for the uncorrelated case\n",
```

Comment:
It might be worth noting that this cell needs to be re-run every time you want to run `reactor_pce_factory.generate_pce` with new settings